Library/Tracking/KSRAnalytics.swift

```diff
@@ -764,9 +763,10 @@ public final class KSRAnalytics {
     case .pledge:
       let allProps = props
         .withAllValuesFrom(optimizelyProperties)
```

Comment:
Ah, sorry, I missed this earlier but can you get rid of the `optimizelyProperties` param. It doesn't look like the new event requires this info.

Reply:
When I got rid of it, this test `testTrackingEvents_Pledge()` in `PledgeCTAContainerViewViewModelTests` fails, because it checks for those `optimizelyProperties`